src/languages/scala.js

```diff
@@ -82,6 +82,11 @@ export default function(hljs) {
         beginKeywords: 'extends with',
         relevance: 10
       },
+      {
+        beginKeywords: 'derives',
+        contains: [ TYPE ],
+        relevance: 10
```

Comment:
Was this needed to resolve a auto-detect conflict?  Otherwise we're really not handing out 10 anymore (relevance is due to change a lot soon with v12 anyways).  So if not, I'd remove it and give it the default relevance.

Reply:
Removed.